src/ai/backend/manager/api/resource.py

```diff
@@ -295,6 +295,322 @@ async def recalculate_usage(request: web.Request) -> web.Response:
     return web.json_response({}, status=200)
 
 
+async def get_stats_for_period(request: web.Request, start_date, end_date, group_ids=None):
```

Comment:
It would be good to relocate this function to session table of model layer.
At least, we can locate them in kernel table.